src/gui/cli/Player.hpp

```diff
@@ -83,8 +83,8 @@ namespace Zappy::GUI {
         private:
             // Attributes
             std::pair<unsigned int, unsigned int> _position;
-            Orientation _orientation;
-            int _level;
+            Orientation _orientation {Orientation::NORTH};
```

Comment:
are you sure about this syntax ? if yes, why not do the same for _inventory ?

Reply:
can't do this on array (tested)